lib/IRGen/GenProto.cpp

```diff
@@ -3732,6 +3732,19 @@ llvm::Constant *IRGenModule::getAddrOfGenericEnvironment(
                           .getIntValue());
         });
 
+        // Need to pad the structure after generic parameters
+        // up to four bytes because generic requirements that
+        // follow expect that alignment.
+        {
+          auto nextOffset = fields.getNextOffsetFromGlobal();
+
+          auto padding = (unsigned)-nextOffset.getValue() & 3;
```

Comment:
Can you replace the 3 with something less magical?

Reply:
I can do `alignment = …` and `padding = alignment - nextOffset % alignment` if that reads better, but there is at least one more please in the code base that does this exact operation with a comment that this is alignment up to 4 bytes…

Reply:
@slavapestov I found a better option - fixed `addAlignmentPadding` and used it instead.

Reply:
That is actually yet another case where `addAlignmentPadding` was used only in one spot that happened to pass pointer alignment as an argument...